wp-content/plugins/tw-posts/tw-posts.php

```diff
@@ -0,0 +1,391 @@
+<?php
```

Comment:
I was able to export the custom post types that affect this Post type into this file.